### PR TITLE
Fwmt 480 bulletproof requests

### PR DESCRIPTION
**Pull request by JacobHarrisonQA**
bullet proofing for whole service

### CODE REVIEW

File: src/test/java/uk/gov/ons/fwmt/fwmtrmadapter/message/impl/JobServiceReceiverImplTest.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.ons.ctp.common.error.CTPException;
 import uk.gov.ons.fwmt.fwmtgatewaycommon.data.DummyTMResponse;
 import uk.gov.ons.fwmt.fwmtgatewaycommon.exceptions.types.FWMTCommonException;
```
[the remark this answers is not in the document]
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/fwmtrmadapter/message/impl/JobServiceReceiverImplTest.java]: done

[the remark this answers is not in the document]
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/fwmtrmadapter/message/impl/JobServiceReceiverImplTest.java]: done

File: src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/config/QueueConfig.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+@Configuration
+public class QueueConfig {
+  private static final String ADAPTER_JOB_SVC_DLQ = "adapter-jobSvc.DLQ";
+  private static final String JOB_SVC_ADAPTER_DLQ = "jobSvc-adapter.DLQ";
+  private static final String RM_ADAPTER_DLQ = "rm-adapter.DLQ";
+  private static final String ADAPTER_RM_DLQ = "adapter-rm.DLQ";
+
+  @Bean
+  public Queue rmToAdapterQueue() {
+    return QueueBuilder.durable(uk.gov.ons.fwmt.fwmtgatewaycommon.config.QueueConfig.RM_TO_ADAPTER_QUEUE)
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/config/QueueConfig.java]: rename uk.gov.ons.fwmt.fwmtgatewaycommon.config.QueueConfig to QueueNames

File: src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/common/retry/CustomMessageRecover.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package uk.gov.ons.fwmt.fwmtrmadapter.common.retry;
+
+import org.springframework.amqp.AmqpRejectAndDontRequeueException;
+import org.springframework.retry.interceptor.MethodInvocationRecoverer;
+
+public class CustomMessageRecover implements MethodInvocationRecoverer<Void> {
+
+  @Override
+  public Void recover(Object[] args, Throwable cause) {
+    System.out.println("IN THE RECOVER ZONE!!!");
```
Comment (spdiaz1000) [inline src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/common/retry/CustomMessageRecover.java]: Remove sysout

**Pull request merged**